glide.yaml

```diff
@@ -0,0 +1,32 @@
+package: github.com/mattermost/mattermost-load-test
```

Comment:
I switched to glide because the kubernetes client has trouble with dep, per https://github.com/golang/dep/issues/1124